fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2116,6 +2116,7 @@ windowFunction
     : name = ROW_NUMBER '(' ')'
     | name = RANK '(' ')'
     | name = DENSE_RANK '(' ')'
+    | name = CUME_DIST '(' ')'
```

Comment:
better add this word to `nonReserved` in `Starrocks.g4`. If not, sql like `select cume_dist from tbl` will fail to parse if `cume_dist` is a col name in tbl.